weave/trace_server/migrations/003_add_display_name.down.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE calls_merged_view MODIFY QUERY
```

Comment:
please include a header and comments on what changes to make it easier for reviewers and readers